### PR TITLE
Fix missing external call instrumentation.

### PR DESCRIPTION
**Pull request by vuqtran88**
This is a fix for #551 .

### CODE REVIEW

File: tests/Agent/IntegrationTests/Shared/WCF/WcfService.cs [hunk 1]
```diff
@@ -62,6 +63,18 @@ public async Task<string> TAPGetData(int value)
             return await DoWork(value.ToString(), false, false);
         }
 
+        public async Task<string> TAPMakeExternalCalls()
+        {
+            using (var httpClient = new HttpClient())
+            {
+                _ = await httpClient.GetAsync("https://google.com");
+                _ = await httpClient.GetAsync("https://bing.com");
+                _ = await httpClient.GetAsync("https://yahoo.com");
```
Comment (JcolemanNR) [inline tests/Agent/IntegrationTests/Shared/WCF/WcfService.cs:72]: Do we care about success or failure of these requests? or is that unimportant?

File: tests/Agent/IntegrationTests/Shared/WCF/WcfClient.cs [hunk 2]
```diff
@@ -189,6 +189,12 @@ public Task<string> TAP_TAPGetData(int value)
 
         }
 
+        public Task<string> TAP_TAPMakeExternalCalls()
+        {
+            return Channel.TAP_TAPMakeExternalCalls();
+
```
Comment (JcolemanNR) [inline tests/Agent/IntegrationTests/Shared/WCF/WcfClient.cs:195]: nitpicky: extra newline

**Review by JcolemanNR — DISMISSED** [review]
needs a changelog update

File: tests/Agent/IntegrationTests/Shared/WCF/WcfService.cs (hunk 1 above)
Comment (tehbio) [reply tests/Agent/IntegrationTests/Shared/WCF/WcfService.cs:72]: It's unimportant - we just need to validate that the instrumentation of the calls happened, even if the requests failed it's not significant for this test.

**Review by JcolemanNR — COMMENTED** [review]
Looks good!

**Review by JcolemanNR — APPROVED** [review]
Looks good!

**Pull request merged**